packages/server/src/__tests__/integration/apolloServerTests.ts

```diff
@@ -2279,7 +2280,9 @@ export function defineIntegrationTestSuiteApolloServerTests(
           const uri = await createServerAndGetUrl({
             gateway,
             documentStore: withDocumentStore
-              ? new KeyvLRU<DocumentNode>()
+              ? new Keyv<DocumentNode>({
+                  store: new LRUStore({ maxSize: 2000 }),
```

Comment:
default store might be fine for this test

Reply:
[15c5326](https://github.com/apollographql/apollo-server/pull/6154/commits/15c53268953f77f9967997e90a931eeb8beca7a4)